libs/legi-shared/date-picker/date-picker.component.ts

```diff
@@ -50,6 +50,11 @@ export class DatePickerComponent extends ControlComponent<Date | null> implement
     return this.config.appearance === 'classic';
   }
 
+  get iconPosition(): 'prefix' | 'suffix' {
+    //for urban appearance, icon is always placed as a suffix
+    return this.config.appearance === 'urban' ? 'suffix' : this.config.datePicker?.iconPosition ?? 'prefix';
```

Comment:
I would suggest something like this, so we can override the position in any theme 🤔 
```suggestion
    if(this.config.datePicker?.iconPosition) {
      return this.config.datePicker.iconPosition
    }
    return this.config.appearance === 'urban' ? 'suffix' : 'prefix';
```

Reply:
done